lib/plugins.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const plugins = {
+  addFlashMessagesToContext: require('./plugins/add-flash-messages-to-context'),
+  error: require('./plugins/error'),
```

Comment:
`error` is a bit too general. Maybe change that to something more specific like `httpErrorHandler` or similar?